docs/DOCUMENTATION.md

```diff
@@ -15,3 +15,7 @@ You can use prop-types to document the intended types of properties passed to co
 
 Mirage.js is a client-side JavaScript library that allows developers to create mock APIs for front-end applications, particularly those built with React. It is often used during the development phase to simulate a back-end server when the actual back-end is not yet fully implemented or available.The primary purpose of Mirage.js is to enable front-end developers to work independently on their applications without being dependent on the back-end team.
 You can add your routs in App.js->createServer->routs function.
+
+#### Material UI framework installed
+
+Material UI has multiple sample to bootstrap projects and applied conventional styling practice
```

Comment:
At the time this branch was for a particular ticket to build a skeleton sign in and sign up form with a header, but since I got behind and there was the package-lock file issue I believed the initial front-end elements are taken care of.

I will need guidance and feedback on what I think would be the best path forward, currently I'm working on github actions but I believed I don't have the updated main branch, so what I'm thinking doing is go back to the remote main and making sure the local branch is update it, then I will complete github actions ticket and then I will check with front-end to see the elements progress, I'm assuming this branch will be outdated by then so it won't be helpful.

Reply:
So while things will be merged to the main branch due to other work going on in the repo; you can always update your local feature branch to have the latest changes from main with the following steps in your terminal:

```
# Run the following steps on your local machine while on your feature branch (eg: you should be in the `craco-install` branch)

# 1. stash any currently work you have going on locally (video for doing it in vscode UI: https://www.youtube.com/watch?v=N6uEF6I4svc)
git stash save

# 2. fetch the changes from the remote (Github) `main` branch AND merge them into your local branch (see screenshot below for how to do it in VS Code)
git pull origin master

# 3. Check if there are any merge conflicts you need to resolve:
git status

# 3b. If git status says there's files with merge conflicts (usually will be coloured in Red) then go to those files and clean up the merge conflicts. Then commit all these changes.

# 4. Push to Github remote feature branch (eg `origin/craco-install`)
git push

# 5. Unstash the changes you were working on so that you can continue developing
git stash pop
```

<img width="960" alt="Screenshot 2023-08-10 at 12 03 16 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/51673cb3-655b-4f92-87a0-669bb9341c50">
^ select "Pull from" in the menu and type in `origin/main`

Reply:
In terms of tickets, if you don't pick up a ticket in JIRA someone may not realize that you were working on it in parallel and may have picked it up and completed it in the meanwhile. Jira is our main tool for tracking who is working on what for this project. 

If the work has already been completed or you want to restart from a new branch, then you can close this pull request with the "Close pull request" button near the bottom.